src/Apps/Artist/Components/ArtistHeader/ArtistHeaderImage.tsx

```diff
@@ -27,71 +27,79 @@ export const ArtistHeaderImage: FC<React.PropsWithChildren<
   const mobile = resized(image.src, {
     width: MOBILE_SIZE.width,
     height: MOBILE_SIZE.height,
-    quality: 60,
+    quality: 50,
   })
 
+  const isMobile = getENV("IS_MOBILE")
+
   return (
     <>
-      <Link
-        rel="preload"
-        href={desktop.src}
-        as="image"
-        imagesrcset={desktop.srcSet}
-        media={`(min-width: ${BREAKPOINTS.sm}px)`}
-      />
+      {isMobile ? (
+        <>
+          <Link
+            rel="preload"
+            href={mobile.src}
+            as="image"
+            imagesrcset={mobile.srcSet}
+            fetchPriority="high"
```

Comment:
With priority LCP image preloading, important to flag as highPriority to boost it up the queue.